gremconnect/websocket.go

```diff
@@ -54,8 +54,8 @@ type WebSocket struct {
 func (ws *WebSocket) Connect() error {
 	var err error
 	dialer := websocket.Dialer{
-		WriteBufferSize:  1024 * 8, // Set up for large messages.
-		ReadBufferSize:   1024 * 8, // Set up for large messages.
+		WriteBufferSize:  1024 * 256, // Set up for large messages.
+		ReadBufferSize:   1024 * 256, // Set up for large messages.
```

Comment:
pretty sure they are not going to accept  a random increase like this one without reason.
This issue here: https://github.com/northwesternmutual/grammes/issues/30 provides reason. But obviously for this, we need to make this configurable.

Reply:
Should we add another method that takes this as param as input? Otherwise we'll have to modify this signature which might not work for upstream.

Reply:
I think for now, we can let it be. There needs to be a more detailed investigation into why we changed what we changed, and if there is a better way to make those changes, before we can merge this to upstream.

Reply:
i tried this but this needs more parameterizing than it looks so i reverted my changes. i think it's better to open up this issue on grammes and then see how it goes